docs/concepts/what-is-a-temporal-cluster.md

```diff
@@ -51,7 +51,7 @@ The Frontend Service has access to the hash rings that maintain service membersh
 
 Inbound call rate limiting is applied per host and per namespace.
 
-The Frontend service talks to the Matching service, History service, Worker service, the database, and Elasticsearch (if in use).
+The Frontend service talks to the Matching service, History service, Worker service, the database, and ElasticSearch (if in use).
```

Comment:
```suggestion
The Frontend service talks to the Matching service, History service, Worker service, the database, and Elasticsearch (if in use).
```
Yeah, my eye keeps jogging on "Elasticsearch" too. However, it's a trademark of Elasticsearch B.V., and that's how they style it, so that's what we use.